### DISCUSSION TITLE
Welcome to lanenet-lane-detection Discussions! (欢迎大家一起讨论相关问题)

### DISCUSSION THREAD

**Discussion by MaybeShewill-CV** [MaybeShewill-CV/lanenet-lane-detection#475] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nurgeldeon** [comment]
Hello, first of all thanks for a great project and novelty in autonomous vehicle field. I have been using this repository for my project; however, currently I am facing an unexpected issue regarding detection. The detection properly works with TuSimple images (both train and test images) but the lane detection fails when I use my own image taken by different camera in a different environment (but it is worth to mention that road lane markings are clearly visible and image size is exactly the same as TuSimple). Also, when I am running the model both binary lane segmentation and pixel embedding generates adequate outputs but afterwards the clustering and drawing pixel coordinates on output image fails. Long story short I am wondering if you might know what can be problem in this case and are there any possible solutions? Thanks!
![lanenet](https://user-images.githubusercontent.com/84007811/117778888-68bf0e00-b23e-11eb-9735-74bd96685312.png)

**Reply by MaybeShewill-CV** [reply]
@nurgeldeon The post process procedure here was designed for Tusimple dataset which may not be suitable to your local dataset. You may either modify the post process code to fit your dataset or you may simply do cluster without post process.

**Comment by ghost** [comment]
Hi. Can someone help me with running the lanedetection model for testing. where is the ckpt for the model exactly? When i try to run it, it says its unable to open the files you provided from the readme.
